x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/IndexLifecycleMetadata.java

```diff
@@ -182,6 +182,11 @@ public String getWriteableName() {
             return TYPE;
         }
 
+        @Override
+        public Version getMinimalSupportedVersion() {
+            return Version.CURRENT.minimumCompatibilityVersion();
```

Comment:
In 7.16 `IndexLifecycleMetadata#getMinimalSupportedVersion()` returns `Version.V_6_6_0`. Obviously this is earlier than `Version.CURRENT.minimumCompatibilityVersion()` but still it would be good to return the same value in both places when backporting.